### PR TITLE
[docker][cygnus-ngsi][feature][NGSIPostgreSQLSink] configure "postgresql_database" in docker

### PR DESCRIPTION
**Pull request by nmatsui**
NGSIPostgreSQLSink can use the user own database by using the "postgresql_database" parameter. But the cygnus-ngsi docker container cannot set "postgresql_database" through an environment variable.
So I modified some docker related files to be able to configure "postgresql_database" by using an environment variable.

### CODE REVIEW

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md [hunk 1]
```diff
@@ -282,9 +282,10 @@ As seen above, the default configuration distributed with the image is tied to c
 * PostgreSQL:
     * It only works for building historical context data in PostgreSQL.
     * The endpoint for PostgreSQL is `iot-postgresql` but can be changed through the CYGNUS_POSTGRESQL_HOST environment variable.
-    * The port for PostgreSQL is `3306` but can be changed through the CYGNUS_POSTGRESQL_PORT environment variable.
+    * The port for PostgreSQL is `5432` but can be changed through the CYGNUS_POSTGRESQL_PORT environment variable.
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md:285]: I understand this (and the other similar change in the file) is a typo fix, right?

File: docker/cygnus-ngsi/cygnus-entrypoint.sh [hunk 2]
```diff
@@ -596,6 +596,9 @@ elif [ "$CYGNUS_POSTGRESQL_HOST" != "" ]; then
     sed -i '/'${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_username/c '${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_username = '${CYGNUS_POSTGRESQL_USER} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     sed -i '/'${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_password/c '${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_password = '${CYGNUS_POSTGRESQL_PASS} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     # The following are optional and disabled by default
+    if [ "$CYGNUS_POSTGRESQL_DATABASE" != "" ]; then
```
Comment (fgalan) [inline docker/cygnus-ngsi/cygnus-entrypoint.sh:599]: Maybe the new env var could be mentioned in the CHANGES_NEXT_RELEASE file. Something like:

```
- Fix: new env var CYGNUS_POSTGRESQL_DATABASE to set Postgresql database in Docker
```

Comment (fgalan) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh:599]: In addition, maybe the new env var should be included in doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md documentation? (Not fully sure...)

Comment (IvanHdzC) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh:599]: I agree, It should be included in the install_with_docker doc.

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md (hunk 1 above)
Comment (nmatsui) [reply doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md:285]: yes, I fixed typo.

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md [hunk 3]
```diff
@@ -282,9 +282,10 @@ As seen above, the default configuration distributed with the image is tied to c
...
     * The user for PostgreSQL is `postgresql` but can be changed through the CYGNUS_POSTGRESQL_USER environment variable.
     * The pass for PostgreSQL is `postgresql` but can be changed through the CYGNUS_POSTGRESQL_PASS environment variable.
+    * The database for PostgreSQL is `postgres` but can be changed through the CYGNUS_POSTGRESQL_DATABASE environment variable.
```
Comment (nmatsui) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md:288]: added a sentence to describe `CYGNUS_POSTGRESQL_DATABASE`.

**Comment by nmatsui** [comment]
Thank you for your review. I updated the `install_with_docker.md` to describe `CYGNUS_POSTGRESQL_DATABASE`, and add a row to `CHANGES_NEXT_RELEASE`.

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md (hunk 1 above)
Comment (fgalan) [reply doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md:285]: Ok, thanks. NTC.

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md (hunk 3 above)
Comment (fgalan) [reply doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md:288]: Ok. Thanks! NTC

File: docker/cygnus-ngsi/cygnus-entrypoint.sh (hunk 2 above)
Comment (fgalan) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh:599]: Fixed in b475f0e

**Comment by AlvaroVega** [comment]
@nmatsui could you please solve conflicts in CNR? Thanks in advance.

**Comment by nmatsui** [comment]
I solved the conflict of CHANGES_NEXT_RELEASE.

**Pull request merged**